common/lib/protocol-definitions/src/sockets.ts

```diff
@@ -59,6 +59,12 @@ export interface IConnect {
      * If we have standardized features across all services, they need to be exposed in more structured way.
      */
     supportedFeatures?: Record<string, any>;
+
+    /**
+     * Properties that client can send to server to tell info about client environment like loader, driver versions etc.
+     * Format: "prop1:val1;prop2:val2;prop3:val3"
+    */
+    relayServiceAgent?: string,
```

Comment:
This is not service agent, this is user agent, i.e. props coming from user to service.
IConnected.relayServiceAgent is right name, as it's communication down (from service to client)